salesforce/10.18/modules/ROOT/pages/salesforce-connector-reference.adoc

```diff
@@ -82,7 +82,7 @@ Uses a username and a password for authentication.
 | Username a| String |  Username used to initialize the session. |  | x
 | Password a| String |  Password used to authenticate the user. |  | x
 | Security Token a| String |  User's security token. It can be omitted if your IP is allowlisted on Salesforce. |  |
-| Authorization URL a| String |  Web service URL responsible for user authentication. This is the URL for the endpoint that is configured to handle SOAP authentication requests. It defaults to the URL containing the latest API version supported by the connector, for example, `+https://login.salesforce.com/services/Soap/u/57.0+`. |  |
+| Authorization URL a| String |  Web service URL responsible for user authentication. This is the URL for the endpoint that is configured to handle SOAP authentication requests. It defaults to the URL containing the latest API version supported by the connector, for example, `+https://login.salesforce.com/services/Soap/u/58.0+`. |  |
```

Comment:
Suggestion: 
URL that is configured > URL configured

URL containing > URL that contains (better not to use gerunds in this scenario - sometimes they can make things less clear)